game-core/src/main/java/games/strategy/triplea/attachments/UnitAttachment.java

```diff
@@ -250,6 +253,22 @@ private void resetCanIntercept() {
     m_canIntercept = false;
   }
 
+  public void setRequiresAirBaseToIntercept(final String value) {
+    m_requiresAirBaseToIntercept = getBool(value);
+  }
+
+  public void setRequiresAirBaseToIntercept(final Boolean value) {
+    m_requiresAirBaseToIntercept = value;
+  }
+
+  public boolean getRequiresAirBaseToIntercept() {
```

Comment:
Not positive, but it looks like only the getter is used outside this class, and all the other methods could be private.

Reply:
Done.